packages/@azure/servicebus/data-plane/test/README.md

```diff
@@ -32,7 +31,7 @@
 
     The environment variables can be set by adding a file by the name `.env` in the root folder of this project.
     Following is a sample .env file template that can be re-used for your environment:
-    ```
+    ```typescript
```

Comment:
The code insider is not really typescript, any reason to add this?

Reply:
**Current `master`**
```
    SERVICEBUS_CONNECTION_STRING=
    
    QUEUE_NAME=partitioned-queue
    QUEUE_NAME_NO_PARTITION=unpartitioned-queue
    QUEUE_NAME_SESSION=partitioned-queue-sessions
    QUEUE_NAME_NO_PARTITION_SESSION=unpartitioned-queue-sessions
    
    TOPIC_NAME=partitioned-topic
    TOPIC_NAME_NO_PARTITION=unpartitioned-topic
    TOPIC_NAME_SESSION=partitioned-topic-sessions
    TOPIC_NAME_NO_PARTITION_SESSION=unpartitioned-topic-sessions
    SUBSCRIPTION_NAME=partitioned-topic-subscription
```
**After updated with `typescript` tag**
```typescript
    SERVICEBUS_CONNECTION_STRING=
    
    QUEUE_NAME=partitioned-queue
    QUEUE_NAME_NO_PARTITION=unpartitioned-queue
    QUEUE_NAME_SESSION=partitioned-queue-sessions
    QUEUE_NAME_NO_PARTITION_SESSION=unpartitioned-queue-sessions
    
    TOPIC_NAME=partitioned-topic
    TOPIC_NAME_NO_PARTITION=unpartitioned-topic
    TOPIC_NAME_SESSION=partitioned-topic-sessions
    TOPIC_NAME_NO_PARTITION_SESSION=unpartitioned-topic-sessions
    SUBSCRIPTION_NAME=partitioned-topic-subscription
```
This looks more readable.
What do you suggest ?

Reply:
you could use `batch` which has less chance of causing confusion. I like the highlighting but I'm worried it'll mislead some people into thinking that needs to go in a ts/js file someplace.

Reply:
I don't see any difference with batch either.
Removed typescript formatting for now.